var/spack/repos/builtin/packages/bcl2fastq2/package.py

```diff
@@ -27,6 +28,9 @@ class Bcl2fastq2(Package):
     # mirror/bcl2fastq2/bcl2fastq2-2.17.1.14.zip and go from there.
     version('2.17.1.14', '7426226c6db095862e636b95c38608d3')
 
+    # Various malloc.h and etc requirements break build on macs
+    conflicts('platform=darwin')
```

Comment:
you can give this a msg attribute and print the comment directly to the user

Reply:
I learn something new every day....